templates/telemetry/bhr/bhr.4.schema.json

```diff
@@ -1 +1,83 @@
-@COMMON_PLACEHOLDER_1_JSON@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "event",
```

Comment:
This is a good place to add a description field linking out to the bug where this is first implemented. The title should probably be `bhr`?